testdata/p4tc_samples_outputs/default_action_example_control_blocks.c

```diff
@@ -224,7 +220,16 @@ if (/* hdr->ipv4.isValid() */
 ;        if (hdr->ipv4.ebpf_valid) {
             outHeaderLength += 160;
         }
-;        pkt = ((void*)(long)skb->data);
+;
```

Comment:
empty statement. can be removed?

Reply:
This is coming from deparser code in ebpf and cant be removed according to design.